CODEOWNERS

```diff
@@ -52,7 +52,7 @@
 /Detectors/Passive                 @sawenzel
 /Detectors/TOF                     @noferini 
 /Detectors/TPC                     @davidrohr @wiechula @shahor02
-/Detectors/TRD                     @jolopezl @bazinski 
+/Detectors/TRD                     @jolopezl @bazinski @tdietel
```

Comment:
Yes. This needs to be done. @ktf: How do we make sure that new people added here (or via groups implicetly) have the write status to the repo? Can this be scripted or at least checked automatically?

Reply:
I would just like to add, that I am not sure if we really have write status. The only new thing I can do now is to ask for reviews to specific people. But I can't merge.

<img width="760" alt="Screen Shot 2020-03-06 at 11 06 56" src="https://user-images.githubusercontent.com/3409812/76090654-b71db080-5f9a-11ea-9d45-836d18a52b55.png">

Reply:
@jolopezl : Having (fake) write status and having merge rights are different things. The CODEOWNERS system (at least for now) is to trigger notifications to experts and to ask for approval. This is at least as far as I understand it.

Reply:
@tdietel has the correct rights.